functions/Test-DbaLastBackup.ps1

```diff
@@ -133,6 +141,7 @@ Restores data and log files to alternative locations and only restores databases
 		[switch]$VerifyOnly,
 		[switch]$NoCheck,
 		[switch]$NoDrop,
+		[switch]$CopyDestination,
```

Comment:
First off, awesome job! The PR went into the proper branch, which is often a challenge. People are really going to appreciate this functionality, thank you.

I like the switch but I'm wondering if people will want to specify custom paths. but then that'd kind of be a pain in the butt. What if! You do -CopyDestination and -CopyPath and if they dont specify CopyPath then default to Backup Dir?